app/styles/ui/dialogs/_rebase-conflicts.scss

```
@@ -0,0 +1,124 @@
+@import '../../mixins';
+
+dialog#rebase-conflicts-list {
+  width: 500px;
+
+  .summary {
+    margin-bottom: 20px;
+  }
+
+  .dialog-header h1 {
+    font-weight: var(--font-weight-light);
+  }
+
+  .dialog-content {
+    padding-bottom: var(--spacing);
+  }
+
+  .green-circle {
+    background-color: var(--color-new);
+    color: var(--background-color);
+    border-radius: 50%;
+    height: 22px;
+    width: 22px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  ul {
+    list-style: none;
+    padding: 0;
+    max-height: 285px;
+    overflow-y: auto;
+
+    li {
+      margin-bottom: var(--spacing-double);
+      padding-left: 0;
+    }
+
+    li:last-of-type {
+      margin-bottom: calc(var(--spacing) * 2);
+    }
+
+    li.unmerged-file-status-resolved,
+    li.unmerged-file-status-conflicts {
+      display: flex;
+      flex-flow: row nowrap;
+
+      .file-octicon {
+        margin-right: var(--spacing);
+      }
+
+      .column-left {
+        display: flex;
+        flex-flow: column nowrap;
+        align-items: start;
+        max-width: 50%;
+        padding-right: var(--spacing);
+
+        .path-text-component {
+          text-overflow: ellipsis;
+          max-width: 100%;
+        }
+      }
+
+      .action-buttons {
+        margin-left: auto;
+        flex-shrink: 0;
+        flex: 0 1 auto;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      .green-circle:last-child {
+        margin-left: auto;
+        margin-top: var(--spacing);
+        flex-shrink: 0;
+        flex: 0 1 auto;
+      }
+    }
+
+    .unmerged-file-status-resolved .file-conflicts-status {
+      color: $green;
+    }
+
+    .unmerged-file-status-conflicts {
+      .file-conflicts-status {
+        color: $orange;
+      }
+
+      .command-line-hint {
+        color: $gray;
+      }
+    }
+  }
+
+  .all-conflicts-resolved {
+    display: flex;
+    flex-flow: row nowrap;
+    padding: var(--spacing) 0 var(--spacing-double);
+
+    .message {
+      padding-left: var(--spacing);
+      padding-top: var(--spacing-third);
+    }
+  }
+
+  .cli-link {
+    a {
+      color: $blue;
+
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+  }
+
+  .arrow-menu {
+    .octicon {
+      width: 10px;
```

If we're ever setting an explicit width on an octicon we also need to set the height or else vertical alignment will be off. Normally we should never set the width but rather use the height and have the width be automatically calculated using the correct aspect ratio. This icon, unfortunately, does not scale cleanly to 10px width though so we should really add a custom vector here but for now let's just fix the vertical alignment by adding a height.
```suggestion
      width: 10px;
      height: 14px;
```

---

I'll also check whether `app/styles/ui/dialogs/_merge-conflicts.scss` needs this change too